application/lib/ui/start_screen_ui.dart

```diff
@@ -1,52 +1,58 @@
 import 'package:flutter/material.dart';
-
 import '../utils/colors.dart';
+import '../utils/image_strings.dart';
 import '../utils/styles.dart';
+import '../utils/text_strings.dart';
+import '../utils/button_style.dart';
 
-class MyApp extends StatelessWidget {
-  const MyApp({Key? key}) : super(key: key);
-
-  @override
-  Widget build(BuildContext context) {
-    return MaterialApp(
-      home: StartScreen(),
-    );
-  }
-}
-class StartScreen extends StatefulWidget {
-  @override
-  _StartScreen createState() => _StartScreen();
-}
-
-class _StartScreen extends State<StartScreen>{
+class StartScreen extends StatelessWidget {
+  const StartScreen({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Center(
-        child: Container(
-          padding: const EdgeInsets.fromLTRB(59.0, 150.0, 55.0, 0.0),
-          color: MyColors.backgroundColor,
-          child: Column(
-            children: [
-              Row(
-                  children:[
-                    Image(
-                      image: AssetImage(welcomeImage),
-                      height: 160,
-                    ),
-                    const Text(
-                      'eventify',
-                      style: Styles.nameStyles,
-                    ),
-                  ]
-              ),
-            ],
+        backgroundColor: MyColors.backgroundColor,
+        body: Center(
+          child: Container(
+            padding: const EdgeInsets.all(10),
+            margin: const EdgeInsets.only(
+                left: 50, top: 250, right: 50, bottom: 90),
+            child: Column(
+              children: [
+                Row(children: [
+                  Image(
+                    image: AssetImage(welcomeImage),
+                    height: 160,
+                  ),
+                  Text(
+                    appName,
+                    style: Styles.nameStyles,
+                  ),
+                ]),
+                const Spacer(
+                  flex: 2,
+                ),
+                Text(
+                  welcomeTitle,
+                  style: Styles.welcomeTitleStyles,
+                  textAlign: TextAlign.center,
+                ),
+                Text(
+                  welcomeSubTitle,
+                  style: Styles.welcomeSubTitleStyles,
+                  textAlign: TextAlign.center,
+                ),
+                const Spacer(
```

Comment:
Ha jól sejtem a Spacer használatával nagyobb távolságot akartál köztük hagyni. Ennek a megvalósítására nem a Spacer, hanem a Padding widgetet szoktuk használni. A másik fontos dolog, hogyha valaminek megadod a flex paraméterét, akkor minden elemnek meg kell adni. A flex paraméter lényege, hogyha a flexek összege mondjuk 6 és az egyik widgetnek beállítod a flex paraméterét 2-re, akkor az a widget a teljes képernyő 2/6-át fogja kitölteni. Mintha százalékokkal adnád meg a szélességet/magasságot. 
Kérlek cseréld ki a Spacer-eket Padding-ekre.